js/render/vertex_array_object.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var assert = require('assert');
+
+module.exports = VertexArrayObject;
+
+function VertexArrayObject() {
+    this.boundProgram = null;
+    this.boundVertexBuffer = null;
+    this.boundVertexBuffer2 = null;
+    this.boundVertexOffset = null;
+    this.boundElementBuffer = null;
+    this.vao = null;
+}
+
+var reported = false;
+
+VertexArrayObject.prototype.bind = function(gl, program, vertexBuffer, vertexBuffer2, vertexOffset, elementBuffer) {
```

Comment:
Can we be consistent in referring to this as either `vertexBuffer2` or `secondVertexBuffer`? (I'm partial to `vertexBuffer2` but not strongly opinionated on this issue.)